src/m365/teams/commands/team/team-add.ts

```diff
@@ -89,31 +89,32 @@ class TeamsTeamAddCommand extends GraphCommand {
         };
         requestBody.description = args.options.description;
       }
-
     }
     else {
       requestBody = {
-        'template@odata.bind': `https://graph.microsoft.com/beta/teamsTemplates('standard')`,
+        'template@odata.bind': `https://graph.microsoft.com/v1.0/teamsTemplates('standard')`,
         displayName: args.options.name,
         description: args.options.description
       }
     }
 
     const requestOptions: AxiosRequestConfig = {
-      url: `${this.resource}/beta/teams`,
+      url: `${this.resource}/v1.0/teams`,
       headers: {
-        accept: 'application/json;odata.metadata=none',
-        'content-type': 'application/json;odata.metadata=none'
+        'accept': 'application/json;odata=nometadata'
```

Comment:
odata=nometadata is not a valid Graph value. Graph requires OData 4 values like odata.metadata=none